.github/workflows/python-integration.yml

```diff
@@ -79,11 +79,33 @@ jobs:
           python -m pip install -e "s3torchconnectorclient[test,e2e]"
           python -m pip install -e "s3torchconnector[test,e2e]"
 
-      - name: s3torchconnector integration tests
-        run: pytest s3torchconnector/tst/e2e -n auto
+      - name: s3torchconnector S3 integration tests
```

Comment:
Should we add s3 express/standard to our matrix? That would decrease duplication here.